Front/pharmacy/src/App.vue

```diff
@@ -3,6 +3,7 @@
 			<Header />
 		<router-link to="/">Home</router-link> |
 		<router-link to="/about">About</router-link>
+			<router-link to="/patients">Patients</router-link>
```

Comment:
Ovo ne ide tu, vec u specifican header. Home i about su obrisani, samo nisu spojeni.

Reply:
da, to sam dodala samo da vidim da li ce se prebaciti na tu stranicu  kad kliknem na to